### DISCUSSION TITLE
followthemoney.org Entity IDs for People

### DISCUSSION THREAD

**Discussion by ColbyReed** [openstates/issues#359] [Proposals & RFCs]
I believe it would be useful to allow the linking of followthemoney.org Entity IDs to People. Possibly under the IDs section like how social media handles are handled.

This would be useful to take advantage of the followthemoney.org api as mentioned here:
https://www.followthemoney.org/our-data/apis

The Sunlight Foundation had a similar project with their "Influence Explorer" a few years back that also used data from followthemoney.org, but that project seems to now be depreciated.

**Comment by jamesturk** [comment]
I'm all for this, the schema does support other_identifiers, so we'd just need to pick a identifier scheme name for them.  

followthemoney or nimsp would make sense.

**Reply by ColbyReed** [reply]
followthemoney may be easier for non dev contributors, in my opinion.

**Reply by ColbyReed** [reply]
They seem to refer to themselves as NIMP around the website, so that might make more sense. Such as here:
https://www.followthemoney.org/our-data/terms-of-data-use

**Reply by jamesturk** [reply]
ah they dropped the S :)  followthemoney works for me
